### ISSUE TITLE
Add filter tick-boxes under the search field

### ISSUE THREAD

**Issue by dietmarw** [impact/search#23]
Once the index supports this it would be nice to have tick boxes under the search.

E.g.,

Search for ...                          
Show:  &#x274f; MA libs       &#x274f; 3rd-party libs  &#x274f; active

**Comment by xogeny** [comment]
Again, this information needs to be in the index before we can represent this in the UI.
